obspy/clients/seedlink/basic_client.py

```diff
@@ -340,7 +340,9 @@ def _packet_handler(self, count, slpack):
             return False
         elif type_ == SLPacket.TYPE_SLINFT:
             if self.debug:
-                print("Complete INFO:" + self.slconn.get_info_string())
+                msg = ("Complete INFO:" +
+                       self._slclient.slconn.get_info_string())
+                print(msg)
```

Comment:
Does it need a space after `:`? In which case, I would do:
```suggestion
                print("Complete INFO:",
                      self._slclient.slconn.get_info_string())
```

Reply:
Ah yeah, was in autopilot mode. This is just a print, I like setting a variable when showing a warning or raising an exception so that the message doesnt show up twice in output..